cves/kernel/CVE-2020-27152.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  An issue was discovered in the Kernal-based virtual machines (KVM) module of the Linux Kernal
+  where an infinite loop would arise due to improper interaction between a resampler and edge 
```

Comment:
Readers might not know what a resampler and edge triggering are. You might want to use less technical language to refer to them, describing them in terms that will allow unfamiliar readers to understand what they are and what they do.